### ISSUE TITLE
"raw_timestamp" list index out of range issue

### ISSUE THREAD

**Issue by kollerbud** [mariostoev/finviz#138]
module was working fine until recently, have been getting this error:

  File "~.\lib\site-packages\finviz\main_func.py", line 110, in get_news
    raw_timestamp = row.xpath("./td")[0].xpath("text()")[0][0:-2]
IndexError: list index out of range

I am assuming something was changed on the website and now this is happening.

**Comment by HunkieCodie** [comment]
Yes, I also noticed this error on a remote machine of mine but it works well on my local machine too.. I'm not sure what the problem might be

**Comment by mariostoev** [comment]
Fixed in v2.0.0. Please upgrade: `pip install --upgrade finviz`
